### ISSUE TITLE
Can't connect to github from termux

### ISSUE THREAD

**Issue by Auxilus** [termux/termux-packages#4047]
**Problem description**
I can't connect to github.com, It throws connection refused for some reason, but I  **can** open and use github.com from browser and also connect via curl.

wget/git/ping all return connection refused.

**Steps to reproduce**
tried cloning, pinging a repo on github.com

**Additional information**
I am on android 9(Pie)
Linux localhost 4.14.62-16294683 #1 SMP PREEMPT Tue Jun 25 09:24:36 KST 2019 aarch64 Android

**Comment by xeffyr** [comment]
**Ping works** (note that it is wrapper for system utility):

![Screenshot_20190713-125359_Termux](https://user-images.githubusercontent.com/25881154/61170215-3f63c700-a56e-11e9-9b8b-1e7512270174.png)

**git** and **curl** are working too:
![Screenshot_20190713-125602_Termux](https://user-images.githubusercontent.com/25881154/61170216-438fe480-a56e-11e9-895f-e747a0682e02.png)

![Screenshot_20190713-125741_Termux](https://user-images.githubusercontent.com/25881154/61170218-47bc0200-a56e-11e9-8d8e-bf1eee288623.png)

**Comment by xeffyr** [comment]
> all return connection refused.

Is this started to happen recently ?

**Comment by juanerasmoe** [comment]
I'm new to termux and have similar problem.
I can connect to git and clone repos with https (with username and password), but connections using an ssh key are always timed out (I followed [this guide](https://www.techrepublic.com/article/how-to-install-git-on-android/) for set up). Pings to github don't work either.
I've tried in three different networks to make sure it isn't a network problem but the problem persists. Any guidance?
